package.json

```diff
@@ -10,8 +10,8 @@
   "main": "app/index.js",
   "scripts": {
     "build": "npm run build:css",
-    "build:css": "node-sass --output=app/static/stylesheets --output-style=expanded app/static/stylesheets",
-    "build:css-watch": "node-sass --output=app/static/stylesheets --output-style=expanded --source-map true --watch app/static/stylesheets",
+    "build:css": "sass --load-path=./ app/static/stylesheets",
```

Comment:
`sass` defaults to output in [expanded style](https://github.com/sass/node-sass#outputstyle).
The output directory is optional and the file is written into the same directory.